### ISSUE TITLE
Unable to upgrade without stopping service manually

### ISSUE THREAD

**Issue by issmirnov** [TryGhost/Ghost-CLI#449]
### This issue is a

* [X] Bug Report
* [ ] Feature Request

### Summary

Running `ghost update` will fail with a `ConfigError`:

```
Message: Port '2368' is in use.
Configuration Key(s): server.port
Current Value(s): 2368
```

I have to manually stop the ghost service, update, then restart the service.

### Steps to Reproduce

1. Install ghost as per official instructions
2. Set up systemd script (I'm running ubutnu 16.04.03 LTS)
3. Launch the service
4. Run `ghost update`

In my opinion, this is a bug. If the port is in use, then ghost could check the PID, ask me to stop it (or interactive prompt and stop it for me) and then restart it back up.

### Technical details (will be automatically output by Ghost-CLI if an error occurs):

* OS:  Ubuntu 16.04.03 LTS
* Node Version: v6.5.0
* Ghost-CLI Version:  1.0.3
* Environment: Systemd, nginx
* Command: `ghost update`

Contents of `/etc/systemd/system/ghost.service `

```
[Unit]
Description=Ghost systemd service for blog: blog-ivansmirnov-name
Documentation=https://docs.ghost.org

[Service]
ExecStart=/usr/local/bin/node /usr/local/bin/ghost run
WorkingDirectory=/var/www/ghost
StandardOutput=syslog
StandardError=syslog
SyslogIdentifier=ghost
User=ghost
Group=ghost
Environment="NODE_ENV=production"
Type=simple

[Install]
WantedBy=multi-user.target
```

Contents of `/var/www/ghost/config.production.json`

```
{
  "url": "https://blog.ivansmirnov.name",
  "server": {
    "port": 2368,
    "host": "127.0.0.1"
  },
  "database": {
    "client": "mysql",
    "connection": {
      "host": "localhost",
      "user": "ghost-REDACTED",
      "password": "REDACTED",
      "database": "ghost_production"
    }
  },
  "mail": {
    "transport": "SMTP",
    "options": {
      "service": "Mailgun",
      "auth": {
        "user": "REDACTED",
        "pass": "REDACTED"
      }
    }
  },
  "logging": {
    "transports": [
      "file",
      "stdout"
    ]
  },
  "process": "systemd",
  "paths": {
    "contentPath": "/var/www/ghost/content"
  }
}
```

**Comment by acburdine** [comment]
@issmirnov just out of curiosity, are you running the ghost service manually via `systemctl start ghost_<service>.service`?

There is currently another open issue (#431) that deals with a similar problem - if you run the ghost service manually and _not_ with `ghost start` then there is no way for Ghost-CLI to know whether or not the instance is currently running - thus causing the problems you're running into.

**Comment by acburdine** [comment]
Closed by #607
